ntbs-service/Services/AuthorizationService.cs

```diff
@@ -215,7 +217,17 @@ public async Task<IQueryable<Notification>> FilterNotificationsByUserAsync(Claim
                     )
                 );
             }
-            return notifications;
+            return notifications
+                .Select(n => new NotificationForHomePage

```

Comment:
I probably would have put this mapping in the index page... it doesn't really feel like it belongs in the `AuthorizationService`.

Reply:
Yep, that makes sense, done!